samples/Sample.iOS/Sample.iOS.csproj

```diff
@@ -124,12 +152,15 @@
   <ItemGroup>
     <None Include="Info.plist" />
     <None Include="Entitlements.plist" />
+    <InterfaceDefinition Include="MainStoryboard.storyboard" />
```

Comment:
Consider adding 2 samples.  One with the view added in the storyboard.  One with the view added in code.  Maybe one app with 2 buttons on main screen and a nav controller.